code/switcher.py

```diff
@@ -169,6 +169,10 @@ def switcher_focus(name: str):
         for cur_app in ui.apps():
             if cur_app.name == wanted_app and not cur_app.background:
                 cur_app.focus()
+
+                while cur_app != ui.active_app():
+                    time.sleep(0.1)
```

Comment:
Hmm, I'm not how reliable this will be on linux/Windows right now. Might end up an infinite loop sometimes - at least I've seen indications that apps can hang around on Linux recently? 

Might need a failsafe.